android/kotlin/AmpliApp/app/src/main/java/com/example/ampliapp/App.kt

```diff
@@ -0,0 +1,80 @@
+package com.example.ampliapp
+
+import android.app.Application
+import com.amplitude.ampli.*
+
+class App : Application() {
+    override fun onCreate() {
+        super.onCreate()
+
+        /**
+         * Start by calling ampli.load(this)
+         *
+         * 'ampli' is the default instance of Ampli()
+         */
+
+        /**
+         * When you pull your tracking plan you can use the defaults and call load(this)
+         *
+         * This requires connecting your account via `ampli pull`
+         * which will set you API key in the generated Ampli SDK
+         */
+        // ampli.load(this)
+
+        /**
+         * OR Specify a Ampli.Environment
+         */
+        // ampli.load(this,
+        //     LoadOptions(
+        //         environment = Ampli.Environment.DEVELOPMENT
+        //     )
+        // )
+
+        /** OR Provide a specific Amplitude API key */
+        // ampli.load(this,
+        //     LoadOptions(
+        //         client = LoadClientOptions(apiKey = AMPLITUDE_API_KEY)
+        //     )
+        // )
+
+        /**
+         * OR Use an existing Amplitude instance
+         * requires "import com.amplitude.api.Amplitude"
+         */
+        // val instance = Amplitude.getInstance()
+        // instance.initialize(this, AMPLITUDE_API_KEY)
+        // ampli.load(this,
```

Comment:
To keep the example more concise could we use this format for all load examples
```
ampli.load(this, LoadOptions(
    client = LoadClientOptions(apiKey = AMPLITUDE_API_KEY)
))
```

Reply:
Done